src/libs/nav2_robot/urdf/turtlebot2.yaml

```diff
@@ -0,0 +1,29 @@
+robot:
```

Comment:
Turtlebot files (urdf and yaml) shouldn't be in this repo. If they're needed for testing they should be pulled in at launch time from their packages.

Reply:
Done.